src/endpoint.cpp

```diff
@@ -540,6 +544,10 @@ Endpoint::AcceptState Endpoint::accept_msg(const struct buffer *pbuf) const
         && has_sys_id(pbuf->curr.target_sysid)) {
         return Endpoint::AcceptState::Accepted;
     }
+
+    if(ignore_comp_id && has_sys_id(pbuf->curr.target_sysid)) {
```

Comment:
Please add a comment what this does and add a blank line before the next one (like the other if statements). E.g. "If comp ID should be ignored and this endpoint has the target of message (sysid only): accept"